src/Features/User-Screen/UserSlice.tsx

```diff
@@ -0,0 +1,59 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { UserProps } from "../../Components/User-Screen";
+
+const initialState: UserProps = {
+  username: "Aman Bhatt",
+  profession: "Web Developer",
+  joinedAt: "2 days ago",
```

Comment:
backend will return a dateTime epoch, write a function to convert epoch to string in utils ?